source/getting-started/components/wazuh-dashboard.rst

```diff
@@ -84,16 +75,16 @@ The Wazuh dashboard includes a Ruleset Test tool that can process log messages t
    :title: Ruleset test
    :alt: Ruleset test
 
-The Wazuh dashboard also includes an API Console for users to interact with the Wazuh API. This can be used to manage the Wazuh deployment (e.g., managing server or agent configurations, monitor status and log messages, adding or removing agents, etc.).
+The Wazuh dashboard also includes a Dev Tools for users to interact with the Wazuh API. This can be used to manage the Wazuh deployment (e.g., managing server or agent configurations, monitor status and log messages, adding or removing agents, etc.).
```

Comment:
Yes, it is a console to run API requests. We renamed this because of the app name change. I will revert the change if you consider the usage of `API Console` allows to undestand it.

Reply:
Fixed here 233c522ccf4b6a33adeb39b0076aade8894816c4.